src/openrct2/drawing/LightFX.cpp

```diff
@@ -276,7 +272,7 @@ void lightfx_prepare_light_list()
         // clang-format on
 #    endif // LIGHTFX_UNKNOWN_PART_1
 
-        if (true)
+        if (false)
```

Comment:
This kills the last of the occlusion code from running. Will try and actually fix the occlusion code eventually.